Refit/Refit-WinRT45.csproj

```diff
@@ -12,12 +12,13 @@
     <RootNamespace>Refit</RootNamespace>
     <AssemblyName>Refit</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
-    <TargetPlatformVersion>8.0</TargetPlatformVersion>
-    <MinimumVisualStudioVersion>11</MinimumVisualStudioVersion>
+    <TargetPlatformVersion>8.1</TargetPlatformVersion>
```

Comment:
ReactiveUI as of v7 has dropped `WPA8` and `WP8` as the userbase is small and the projects do not open in Visual Studio. Essentially the path taken for that project is to keep it around (if painless) until Visual Studio (latest) no longer opens the project. So. I'm a-okay with dropping this platform and retargeting.